tutorials/source-modeling/plot_beamformer_lcmv.py

```diff
@@ -286,8 +279,8 @@
 stc_fs = morph.apply(stc)
 del stc
 
-brain = stc_fs.plot_3d(
-    src=src_fs, initial_time=0.085, subjects_dir=subjects_dir,
+stc_fs.plot(
+    src=src_fs, mode='stat_map', initial_time=0.085, subjects_dir=subjects_dir,
     clim=dict(kind='value', pos_lims=lims), verbose=True)
```

Comment:
What does this change achieve? Asking for a friend. 😁

Reply:
`plot` uses matplotlib 2D mode via `nilearn.plotting`, `plot_3d` uses 3D volumetric rendering via `_Brain`

Reply:
thanks!